manifests/init.pp

```diff
@@ -14,12 +14,16 @@
   $write_queue_limit_low  = undef,
   $package_name           = $collectd::params::package,
   $version                = installed,
+  $minimum_version        = undef,
 ) inherits collectd::params {
 
   $plugin_conf_dir = $collectd::params::plugin_conf_dir
   validate_bool($purge_config, $fqdnlookup)
   validate_array($include, $typesdb)
 
+  # Version for templates
+  $_collectd_version = pick($::collectd_version, regsubst($version, '^\d+(?:\.\d+){1.2}', '\0'), $minimum_version)
```

Comment:
what is this regex for ? if that's for sanity, I'd prefer a validate_re so we can fail if the input is incorrect